cfme/intelligence/reports/widgets/__init__.py

```diff
@@ -100,6 +94,44 @@ def delete(self, cancel=False):
             view.flash.assert_no_error()
 
 
+class DashboardReportWidgetsCollection(BaseCollection):
```

Comment:
This needs to be `@attr.s` decorated even though it doesn't define any attributes locally.